src/nix-build/nix-build.cc

```diff
@@ -417,16 +417,19 @@ void mainWrapped(int argc, char * * argv)
                 "dontAddDisableDepTrack=1; "
                 "[ -e $stdenv/setup ] && source $stdenv/setup; "
                 "%3%"
+                "PATH=\"%4%:$PATH\"; "
+                "SHELL=%4%; "
```

Comment:
Probably PATH and SHELL shouldn't both refer to `%4%`?

Reply:
:man_facepalming:

Reply:
```
SHELL=/nix/store/pd3xx55jjik53gvbyrs4h0him03l3wah-bash-interactive-4.4-p23/bin/bash
PATH=/nix/store/pd3xx55jjik53gvbyrs4h0him03l3wah-bash-interactive-4.4-p23/bin:...
```